docs/code/IDataViewImplementation.md

```diff
@@ -159,7 +160,7 @@ method, and you use this delegate multiple times to fetch the actual column
 values as you `MoveNext` through the cursor.
 
 Some history to motivate this: In the first version of `IDataView` the
-`IRowCursor` implementation did not actually have these "getters" but rather
+`DataViewRowCursor` implementation (formerly known as `IRowCursor`) did not actually have these "getters" but rather
```

Comment:
I see one usage of and various references to `IRowCursor` -- https://github.com/dotnet/machinelearning/search?utf8=%E2%9C%93&q=IRowCursor&type=

Example:
https://github.com/dotnet/machinelearning/blob/610ffcb67083c2e5e6e1a14884ba24b1da0384c7/src/Microsoft.ML.Data/Evaluators/BinaryClassifierEvaluator.cs#L1540

Reply:
Yes, it seems it is the only place in "actual code" where `IRowCursor ` appears.

It is wrapped inside a` #if !CORECLR` ([link](https://github.com/dotnet/machinelearning/blob/610ffcb67083c2e5e6e1a14884ba24b1da0384c7/src/Microsoft.ML.Data/Evaluators/BinaryClassifierEvaluator.cs#L1466)). So when building ML.NET nothing inside that block is compiled. If I remove the CORECLR if directive, then I get some errors, not only corresponding to the IRowCursor (which doesn't exist currently in ML.NET).

I don't know if I should change it, then.

Reply:
The code in `BinaryClassifierEvaluator` will not compile even if we change `IRowCursor` to `DataViewRowCursor`, since it uses a class that does not exist in our code: `XYPlot`. We had that class in an internal repo, and it used System.Windows.Forms which I believe is only for .Net Framework, so we didn't include it in ML.NET. I think that code can safely be deleted.

---
In reply to: [368201681](https://github.com/dotnet/machinelearning/pull/4673#discussion_r368201681) [](ancestors = 368201681)

Reply:
I will open another PR to remove that piece of code then, since I prefer that this PR remains only as changes in the documentation. Thanks for the clarification, @yaeldekel !

---
In reply to: [368281931](https://github.com/dotnet/machinelearning/pull/4673#discussion_r368281931) [](ancestors = 368281931,368201681)

Reply:
I've opened the PR in here: [https://github.com/dotnet/machinelearning/pull/4694](https://github.com/dotnet/machinelearning/pull/4694)

---
In reply to: [369266609](https://github.com/dotnet/machinelearning/pull/4673#discussion_r369266609) [](ancestors = 369266609,368281931,368201681)